build_scripts/pg/db_init.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
what is the directory naming convention in elixir? snake_case, camelCase, kebab-case?

Reply:
Elixir uses snake_case as a directory/file naming convention.